views/admin/sendmail/index.tt

```diff
@@ -12,45 +12,60 @@
 %]
 
 [%- sections.title='.title' | ml -%]
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        .error { color: #f04124; }
+    </style>
+[% END %]
 
 <p>[% '.intro' | ml %]</p>
 
+[%- INCLUDE components/errors.tt errors = errors -%]
+
 <form method="POST">
 
 [%- IF has_menu -%]
     [% dw.form_auth %]<p>
-    [%- label_a = ".select.account.label" | ml;
-        form.select( label = label_a, name = "account", id = "account",
+    [%- form.select( label = dw.ml( ".select.account.label" ),
+                     name = "account", id = "account",
                      items = account_menu );
     -%]</p><p>
-    [%- label_t = ".input.sendto.label" | ml;
-        form.textbox( label = label_t, name = "sendto", id = "sendto",
+
+    [%- form.textbox( label = dw.ml( ".input.sendto.label" ),
+                      name = "sendto", id = "sendto",
                       maxlength = "255", size = "50", autocomplete = "off" );
     -%]&nbsp;&nbsp;&nbsp;[% ".input.sendto.note" |ml %]</p><p>
-    [%- label_s = ".input.subject.label" | ml;
-        form.textbox( label = label_s, name = "subject", id = "subject",
+
+    [%- form.textbox( label = dw.ml( ".input.subject.label" ),
+                      name = "subject", id = "subject",
                       maxlength = "255", size = "50", autocomplete = "off" );
     -%]</p><p>
-    [%- label_m = ".input.message.label" | ml;
-        form.textarea( label = label_m, name = "message", id = "message",
+
+    [%- form.textarea( label = dw.ml( ".input.message.label" ),
+                       name = "message", id = "message",
                        rows = '30', cols = '80', wrap = 'soft',
                        style = 'vertical-align: top' );
     -%]</p><p>
-    [%- label_r = ".input.supportreq.label" | ml;
-        form.textbox( label = label_r, name = "request", id = "request",
+
+    [%- form.textbox( label = dw.ml( ".input.supportreq.label" ),
+                      name = "request", id = "request",
                       maxlength = "10", size = "10", autocomplete = "off" );
     -%]&nbsp;&nbsp;&nbsp;
-    [%- label_c = ".check.supportreq.label" | ml;
-        form.checkbox( label = label_c, selected = 1, value = 1,
-                       name = "reqsubj", id = "reqsubj" );
-    -%]</p><p>[%- ".input.notes.label" | ml -%]<br />
+
+    [%- form.checkbox( label = dw.ml( ".check.supportreq.label" ),
+                       name = "reqsubj", id = "reqsubj",
+                       selected = 1, value = 1 );
+    -%]</p><p>
+
+    [%- ".input.notes.label" | ml -%]<br />
```

Comment:
Thnik I missed this the first time -- but please wrap this label in `<label>` tags